core/src/main/python/wlsdeploy/tool/validate/validator.py

```diff
@@ -108,9 +109,11 @@ def validate_in_standalone_mode(self, model_dict, variables_file_name=None, arch
         """
         _method_name = 'validate_in_standalone_mode'
 
+        cloned_model_dict = copy.deepcopy(model_dict)
```

Comment:
Please add a comment here to describe why its necessary to do a deep copy so someone doesn't accidentally change it back.